index2.html

```diff
@@ -23,55 +23,241 @@
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
                 </ul>
+            </nav>
+            
+            <section class="headline">
+                <header class="header">
+                    <h1>Hey there. I’m Franck sefu.<br>
+                    <span>I’m a software developer</span></h1>
+                </header>
+                <section class="content">
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                    </p>
+                </section>
+                <section class="icon">
+                    <ul>
+                        <li>
+                            <a href="https://medium.com/@francksefu1998">
+                                <i class="fab fa-medium"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/franck_sefu">
+                                <i class="fab fa-twitter"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/franck-sefu-884705254/">
+                                <i class="fab fa-linkedin-in"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/francksefu">
+                                <i class="fab fa-github"></i>
+                            </a>
+                        </li>
+                    </ul>
+                    
+                </section>
+                
+                <button><img src="Disabled.png" id="img"></button>
                 
             </section>
+    
+        </section>
+    
+        <!--Build Recent work section-->
+            
+        <section class="recentWork">
+            <div class="rHeader title">
+                <h1 class="title">My Recent<br> Works</h1>
+                <div class="bar"> </div>
+            </div>
+            
             
-            <button><img src="Disabled.png" id="img"></button>
+            <article>
+                <div> </div>
+                <div class="detail">
+                    <div class="rHeader">
+                        <h2>Multi-Post Stories<br> Gain+Glory</h2>
+                    </div>
+                    
+                    <ul class="language">
+                        <li><span>Ruby on rails</span></li>
+                        <li><span>css</span></li>
+                        <li><span>JavaScript</span></li>
+                        <li><span>html</span></li>
+                    </ul>
+                    <button class="buttonCol"><span>See Project</span></button>
+                </div>
```

Comment:
- [ ] While you made good adjustments to the suggestion of the previous reviewer, there are still some improvements needed here to make it pixel-perfect to Figma design.
- [ ] The margin between `see project` button and the border of the card is too much. kindly reduce the space and make it similar to the figma design.
- [ ] kindly consider applying this change in all cards.
your design
![qw](https://user-images.githubusercontent.com/96683843/212470850-566ab6b3-451c-4ff1-a706-9a68b87733d6.jpg)

figma design
![qwer](https://user-images.githubusercontent.com/96683843/212470855-d265d4bf-a0c5-4178-9943-3e9bd54662f9.jpg)